e2e/__tests__/customTestSequencers.test.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import path from 'path';
+import runJest from '../runJest';
+import {extractSummary} from '../Utils';
+const dir = path.resolve(__dirname, '../custom-test-sequencer');
+
+expect.extend({
+  toBeIn(received, arr) {
```

Comment:
core matcher `toContain` does the same thing

Reply:
I think `toContain` is not same as `toBeIn`.
`toBeIn` is expect item in the list of giving array, and it works in `toEqual`.
`toContain` is expect an array contain item, can't work in `toEqual`.

Reply:
we also have `toContainEqual`